common/meta/api/src/meta_api_test_suite.rs

```diff
@@ -955,6 +957,36 @@ impl MetaApiTestSuite {
 
         Ok(())
     }
+
+    pub async fn share_create_get_drop<MT: MetaApi>(&self, mt: &MT) -> anyhow::Result<()> {
+        let tenant1 = "tenant1";
+        let share_name1 = "share1";
+        tracing::info!("--- create {}", share_name1);
+        {
+            let req = CreateShareReq {
+                if_not_exists: false,
+                tenant: tenant1.to_string(),
+                share_name: share_name1.to_string(),
+            };
+
+            let res = mt.create_share(req).await;
+            tracing::info!("create share res: {:?}", res);
+            let res = res.unwrap();
+            assert_eq!(1, res.share_id, "first share id is 1");
```

Comment:
`Create` needs a test case that creates a `share` when the `share` already exists.

Similar for `DropShare` test.

Reply:
If a simple `get_share()` won't bring in too much work, a new feature that is fully tested  would be better:D

Reply:
added 4 commits for get_share and more tests